metaci/release/models.py

```diff
@@ -46,11 +47,22 @@ class Release(StatusModel):
     )
 
     release_creation_date = models.DateField(
-        _("release creation date"), null=True, blank=True
+        _("release creation date"),
+        null=True,
+        blank=True,
+        default=(datetime.date.today().isoformat()),
+    )
+    sandbox_push_date = models.DateField(
+        _("sandbox push date"),
+        null=True,
+        blank=True,
+        default=(datetime.date.today().isoformat()),
     )
-    sandbox_push_date = models.DateField(_("sandbox push date"), null=True, blank=True)
     production_push_date = models.DateField(
-        _("production push date"), null=True, blank=True
+        _("production push date"),
+        null=True,
+        blank=True,
+        default=((datetime.date.today() + datetime.timedelta(days=6)).isoformat()),
```

Comment:
@jofsky You still haven't fixed these so that evaluating the default is deferred until the object is actually being constructed.

You need to define a function like this:

    def get_today():
        return datetime.date.today()

And then use it in the model like this:

    default=get_today,

Also notice I removed `.isoformat()`. It's a DateField, which means we can give it dates to store. No need to convert to a string and then make it convert back.

You will also need to update the database migrations.